dask_bigquery/core.py

```diff
@@ -19,33 +19,29 @@ def bigquery_client(project_id=None):
     See  googleapis/google-cloud-python#9457
     and googleapis/gapic-generator-python#575 for reference.
     """
-
-    bq_storage_client = None
     with bigquery.Client(project_id) as bq_client:
-        try:
-            bq_storage_client = bigquery_storage.BigQueryReadClient(
-                credentials=bq_client._credentials
-            )
-            yield bq_client, bq_storage_client
-        finally:
-            bq_storage_client.transport.grpc_channel.close()
+        bq_storage_client = bigquery_storage.BigQueryReadClient(
+            credentials=bq_client._credentials
+        )
+        yield bq_client, bq_storage_client
+        bq_storage_client.transport.grpc_channel.close()
 
 
-def _stream_to_dfs(bqs_client, stream_name, schema, timeout):
+def _stream_to_dfs(bqs_client, stream_name, schema, read_kwargs):
     """Given a Storage API client and a stream name, yield all dataframes."""
     return [
         pyarrow.ipc.read_record_batch(
             pyarrow.py_buffer(message.arrow_record_batch.serialized_record_batch),
             schema,
         ).to_pandas()
-        for message in bqs_client.read_rows(name=stream_name, offset=0, timeout=timeout)
+        for message in bqs_client.read_rows(name=stream_name, offset=0, **read_kwargs)
     ]
 
 
 def bigquery_read(
     make_create_read_session_request: callable,
     project_id: str,
-    timeout: int,
+    read_kwargs: int,
```

Comment:
@jrbourbeau I was going over the docs and realized that this still shows as an `int`. Shouldn't it go as a keyword argument at the end? `read_kwargs: dict=None`, and I wonder where should the `*` go, right before it? Like 
```python
def bigquery_read(
    make_create_read_session_request: callable,
    project_id: str,
    timeout: int,
    stream_name: str, 
    *, 
    read_kwargs: dict=None)
```
If this is correct I can modify it. and we should probably add a test that this works, although I'm not sure what's the easist to test for these kwargs, any ideas?

Reply:
Since `bigquery_read` is only ever called internally in `read_gbq` I don't think it matters whether or not `read_kwargs` is a positional or keyword argument to `bigquery_read`. Though you bring up a good point that the type annotation is now incorrect and should be updated to `dict` instead of `int`